docs/faqs/other_effects.md

```diff
@@ -9,7 +9,7 @@ to resurrect support for the effect types they use in repository they control.
 
 You can read about the rationale for decision [here](https://github.com/disneystreaming/weaver-test/discussions/570). Feel free to ping us via a github discussion, if you seek to resurrect support for a given effect-type.
 
-If you are looking for documentation of the maintenance branch of weaver that did support other effect types, you can find it [over there](https://disneystreaming.github.io/weaver-test/docs/0.6.15/installation).
+If you are looking for documentation of the maintenance branch of weaver that did support other effect types, you can find it [over there](https://github.com/disneystreaming/weaver-test/blob/v0.6.15/docs/installation.md).
```

Comment:
Assuming the repo is archived and not deleted, we can still reference this.

That said, I wonder if this portion of the docs could be removed entirely